### PR TITLE
[RNBW-1438] Transaction Speed Up / Cancel improvements

### PR DESCRIPTION
**Pull request by derHowie**
fixes RNBW-1438

**Testing Instructions:**
* Tweak `module.exports` in `hardhat.config.js`:
```
// hardhat.config.js
module.exports = {
  networks: {
    hardhat: {
      mining: {
        auto: false,
        interval: 1000 * 1,
      },
    },
  },
  solidity: '0.8.4',
};
```
cases:
* send (eth, token, nft)
* swap
* deposit
* withdraw
* walletconnect

### CODE REVIEW

**Review by brunobar79 — DISMISSED** [review]
Code looks good and tested it myself. Good to merge after CI passing

File: src/entities/transactions/transaction.ts [hunk 1]
```diff
@@ -12,10 +13,11 @@ export interface RainbowTransaction {
     display: string;
   } | null;
   dappName?: string; // for walletconnect
+  data?: string;
```
Comment (jinchung) [inline src/entities/transactions/transaction.ts:16]: can include a comment that this is also `// for pending tx`

**Comment by jinchung** [comment]
Note to include same `data` and `value` for `actions/unlock`

**Review by jinchung — APPROVED** [review]
🍔

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**